simmechanics/data/icub3/ICUB_3_upperbody_options.yaml

```
@@ -19,17 +19,17 @@ rename:
     SIM_ICUB3_L_SHOULDER_2: l_shoulder_2
     SIM_ICUB3_L_SHOULDER_1--SIM_ICUB3_L_SHOULDER_2: l_shoulder_roll
     SIM_ICUB3_L_SHOULDER_3: l_shoulder_3
-    SIM_ICUB3_L_SHOULDER_2--SIM_ICUB3_L_SHOULDER_3: l_shoulder_yaw
+    SIM_ICUB3_L_SHOULDER_2--SIM_ICUB3_L_SHOULDER_3: l_arm_ft_sensorx

```

```suggestion
    SIM_ICUB3_L_SHOULDER_2--SIM_ICUB3_L_SHOULDER_3: l_arm_ft_sensor
```
Is the `x` expected or can it be removed?